src/h5web/visualizations/index.test.ts

```diff
@@ -211,13 +211,16 @@ describe('Visualization definitions', () => {
     });
 
     it('should support group with absolute `default` path to supported NXdata group', () => {
-      const group = makeMyNxEntityGroup('foo', 'NXentry', {
-        defaultPath: '/foo/bar',
-        children: [makeMyNxDataGroup('bar', spectrumDatasetInt1D)],
-        parents: [makeMyNxEntityGroup('root', 'NXroot')],
```

Comment:
In this "bottom-up" syntax, parents were created before children (i.e. in pseudo-code `makeChild(makeParent())`). In #365, I'm using a "top-down" in which children are created before their parents (i.e. in pseudo-code: `makeParent(makeChild())`).

The "bottom-up" syntax was only used in this file and has a simple alternative as seen below. I think it's better to stick to only one syntax, and to favour the syntax that matches the mental model of a tree (i.e. the "top-down" syntax).

Reply:
I would have inverted the names "top-down" and "bottom-up" :joy: 

But yes, the new syntax feels more straightforward to me

Reply:
Haha yeah, can go either way depending on whether we're talking imperatively or declaratively.